### ISSUE TITLE
GPA is not library friendly

### ISSUE THREAD

**Issue by chaoticbob** [GPUOpen-Tools/GPA#36]
GPA usage of CMake is very awkward making it difficult to simply include GPA into an existing large project by setting variables and then calling ```add_subdirectory```. Currently doing so causes the build process to produce binary paths that can't be found.

**Comment by chesik-amd** [comment]
Yes, this isn't a usage model that we considered.  We'll take a look to see what can be done to make this experience better.

**Comment by chesik-amd** [comment]
I've just pushed a few changes to the CMake files. WIth these changes, we've been able to include GPA in another project by simply using `add_subdirectory`. There is still a prerequisite that you'll need to run UpdateCommon.py before doing this.  Can you give the latest sources a try and see if it works better for you? If you still run into problems, we may need more info on exactly how you are using this, and what exact problems you still see.

Thanks!
